src/components/addHotel/AddHotel.js

```diff
@@ -0,0 +1,305 @@
+            <label htmlFor="neighbourhood">
+              Neighbourhood
+              <input
+                id="neighbourhood"
+                type="text"
+                name="neighbourhood"
+                value={values.neighbourhood}
+                onChange={(e) => {
+                  setValues({ ...values, neighbourhood: e.target.value });
+                }}
+              />
+            </label>
+
+          </div>
+
+          <div className="checkbox">
+            <label htmlFor="street">
+              Street
+              <input
+                type="text"
+                name="street"
+                value={values.street}
+                onChange={(e) => {
+                  setValues({ ...values, street: e.target.value });
+                }}
+              />
+            </label>
+
+          </div>
+
+          <div className="checkbox">
+            <label htmlFor="number">
+              Number
+              <input
+                type="number"
+                name="number"
+                value={values.number}
+                onChange={(e) => {
+                  setValues({ ...values, number: e.target.value });
+                }}
+              />
+            </label>
+
+          </div>
+
+          <div className="checkbox">
+            <label htmlFor="complement">
+              Complement
+              <input
+                type="complement"
+                name="complement"
+                value={values.complement}
+                onChange={(e) => {
+                  setValues({ ...values, complement: e.target.value });
+                }}
+              />
+            </label>
+
+          </div>
+        </div>
+
+        <div className="section-form">
+          <h2>Hotel data</h2>
+
+          <div className="checkbox">
+            <label htmlFor="name">
+              Name
+              <input
+                type="text"
+                name="name"
+                value={values.name}
+                onChange={(e) => {
+                  setValues({ ...values, name: e.target.value });
+                }}
+              />
+            </label>
+
+          </div>
+
+          <div className="checkbox">
+            <label htmlFor="description">
+              Description(Min. 20 chars)
+              <input
+                type="text"
+                name="description"
+                value={values.podescriptionol}
+                onChange={(e) => {
+                  setValues({ ...values, description: e.target.value });
+                }}
+              />
+            </label>
+
+          </div>
+
+          <div className="checkbox">
+            <label htmlFor="room">
+              Rooms
+              <input
+                type="number"
+                name="room"
+                value={values.room}
+                onChange={(e) => {
+                  setValues({ ...values, room: e.target.value });
+                }}
+              />
+            </label>
+
```

Comment:
There should be a better way to write this, maybe a looping structure of some sort?